### PR TITLE
Update ci.yml file

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Call `set-test-pipeline-version.yml` in `C:\Git\azure-sdk-for-java\eng\pipelines\templates\jobs\ci.yml`

### CODE REVIEW

File: eng/pipelines/templates/jobs/ci.yml [hunk 1]
```diff
@@ -50,15 +50,11 @@ jobs:
...
-            filePath: eng/scripts/SetTestPipelineVersion.ps1
-            arguments: '-BuildID $(Build.BuildId)'
+      - template: /eng/common/pipelines/templates/steps/set-test-pipeline-version.yml
+        parameters:
+          PackageName: "azure-sdk-template"
+          ServiceDirectory: "template"
+          TestPipeline: ${{ parameters.TestPipeline }}
 
       - script: |
           echo "##vso[build.addbuildtag]Scheduled"
```
Comment (weshaggard) [inline eng/pipelines/templates/jobs/ci.yml:60]: What about the other case? https://github.com/Azure/azure-sdk-for-java/commit/0d95698d3179cff99e96b4a5a7154231ca387ae3#diff-78f6357ab2cf93b8a11c1fb81a2ba78da1e364b1d14be096d9076e0ccfb2b324R186 Do you not need to add that to the Analyze job?

Comment (chidozieononiwu) [reply eng/pipelines/templates/jobs/ci.yml:60]: Right.

**Review by weshaggard — APPROVED** [review]

**Comment by weshaggard** [comment]
test failure is caused by https://github.com/Azure/azure-sdk-for-java/pull/20073 and not related so ignoring it for this PR.

**Pull request merged**